### ISSUE TITLE
Failed iterations count is always zero - even if an iteration had assertion errors

### ISSUE THREAD

**Issue by enorm** [postmanlabs/newman#1600]
Hello there,

Version and environment information:

 1. Newman Version: 3.9.4
 2. OS details: Mac and CentOS
 3. Are you using Newman as a library, or via the CLI? Library
 4. Did you encounter this recently, or has this bug always been there: I think it has always been there
 
**5. Expected behaviour:**

When assertions within an iteration fail, I expect the iteration fail count to increase. Instead, it remains at 0.
This way, I cannot count successful iterations, only "run" iterations. I use the following variables:
* summary.run.stats.iterations.total
* summary.run.stats.iterations.failed

Additionally, the CLI report of test iterations displays this behavior (see attached screen shot): at least one iteration had failed assertions, but the failed iteration count is 0.

![image001](https://user-images.githubusercontent.com/1926623/42040349-ae9487cc-7aef-11e8-9c1e-766f9aa1cf3e.png)

Is this a bug or is my understanding faulty?

Best regards
Norman

**Comment by shamasis** [comment]
Yep. Agreed. This should be addressed. The summary computation should mark failed iteration +1 if any test failed / request failed in that iteration.

**Comment by shamasis** [comment]
@kunagpal - what needs to be changed in runtime for this? can it not be done completely from CLI reporter?

**Comment by gamblegoon** [comment]
i dont understand ,,, what is wrong with my jukebox

**Comment by svarshney2311** [comment]
Is this fixed or still open. I am having same type of issue. even the assertion fails, Test requests does not count.

**Comment by pulkit-30** [comment]
I think I can solve this issue, may you please assign me this issue.
